.mailmap

```diff
@@ -105,3 +105,6 @@ Derrick Dymock <actown@gmail.com> Derrick Dymock <derrick@puppetlabs.com>
 
 # Alias of Joel Kees <joelkees@gmail.com>
 Joel Kees <joelkees@gmail.com> Joeliam <joelkees@gmail.com>
+
+# Alias of Karl Dietz <dekarl@users.sourceforge.net>
```

Comment:
Shouldn't this be the "spaetfruehstuecken" one (assuming that's the new cannonical one)?